lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -191,10 +191,23 @@ def find_or_create_for(application, resource_owner_id, scopes, expires_in, use_r
       # @return [Doorkeeper::AccessToken] array of matching AccessToken objects
       #
       def authorized_tokens_for(application_id, resource_owner_id)
-        ordered_by(:created_at, :desc)
+        # In case of static token expiry times, we can optimize fetching tokens
+        # by already filtering out tokens whose TTL has expired.
+        # We can't do that if custom expirations times have been configured, though.
+        ttl = unless Doorkeeper.configuration.option_defined?(:custom_access_token_expires_in)
```

Comment:
This construction looks complex to me :thinking: assignment with `unless`... Is it possible to simplify it in order to make more human-friendly aka readable?

Reply:
replaced it with assignment inside the unless